packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -999,20 +994,14 @@ export abstract class FluidDataStoreContext
 				safeTelemetryProps,
 			);
 
-			sendGCUnexpectedUsageEvent(
-				this.mc,
+			this.mc.logger.sendTelemetryEvent(
 				{
 					eventName: "GC_Tombstone_DataStore_Changed",
```

Comment:
Any good way to move this log to gcTelemetry? I guess we'd have to pass a `nodeUsed` callback here?

Reply:
I believe we can. Especially with `nodeUsed` now accepting any additional params to log, we could add callsite to distinguish various cases.